scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -272,21 +290,20 @@
   ">
   {% if variant.clinsig_human %}
     <span class="badge bg-secondary">ClinVar</span>
-    <br>
   {% endif%}
   {% if variant.cosmic_ids %}
     <span class="badge bg-secondary">COSMIC</span>
-    <br>
+
   {% endif %}
   {% if variant.mitomap_associated_diseases %}
     <span class="badge bg-secondary">MITOMAP</span>
-    <br>
   {% endif %}
   {% if variant.clingen_cgh_pathogenic %}
-    <span class="badge bg-secondary">CGH Pathogenic</span><br>
+    <span class="badge bg-secondary">CGH Pathogenic</span>
   {% endif %}
   {% if variant.local_obs_old or variant.local_obs_cancer_germline_old or variant.local_obs_cancer_somatic_old %}
-    <span class="badge bg-secondary">Local</span>
+    <span class="badge bg-secondary position-relative">Local{% if variant.category in ['snv', 'sv'] %}<span class="badge bg-info rounded-pill position-absolute top-0 start-100 translate-middle">
```

Comment:
counts...

Reply:
I like this style but I'm afraid the count numbers are not very visible?

<img width="839" alt="image" src="https://github.com/user-attachments/assets/3964a867-11aa-4d81-b6f9-3c9efd7ce1f8">

With dark therme it is slightly better but perhaps add them inside the "Local" badge itself inside a parenthesis?

Reply:
Hm, yes, the info color is really a bit in-between on whether to have light or dark color text on it, contrast wise. Especially with all the white around on light mode, it becomes a bit hard to see. It is still in the tooltip I guess, and it is pretty obvious what order of magnitude the number is. But for magnitude we could do different colors, stars or whatnot instead of fuzzy numbers. Lets see...

Reply:
How about this:
![Screenshot 2024-10-15 at 08 35 41](https://github.com/user-attachments/assets/7fa45c3c-f125-4b9f-bd32-57a06fcbe933)
![Screenshot 2024-10-15 at 08 35 47](https://github.com/user-attachments/assets/9d7029f9-c4f7-48bd-a102-862d5485788a)
When they changed to automatic pairing of bg and text color helpers in B 5.2, they paired dark text with the info badge: https://getbootstrap.com/docs/5.2/components/badge/#background-colors

Reply:
Better! 👍🏻

Reply:
Or maybe the dark is a little worse. Let's go with `text-body`.

Reply:
> Or maybe the dark is a little worse. Let's go with `text-body`.

It was better on light mode tho

Reply:
Yeah, so with body keeping the dark on light, and light on dark we should be good, right?

Reply:
> Yeah, so with body keeping the dark on light, and light on dark we should be good, right?

Or just put it between parentheses in the other badge?

Reply:
Ok, let's make it a badge-in-the-badge then. It's essentially like parenthesis, but without the extra characters.
![Screenshot 2024-10-15 at 10 41 10](https://github.com/user-attachments/assets/c7963c37-2d42-4369-83ae-86b29596eb38)
![Screenshot 2024-10-15 at 10 40 54](https://github.com/user-attachments/assets/3dc98db3-522e-4fa2-9fc3-0387cd24f5c1)